src/core-services/account/util/canAddAccountToGroup.js

```diff
@@ -17,18 +15,9 @@ export default async function canAddAccountToGroup(context, group) {
 
   const { shopId } = group;
 
-  // An account can add another account to a group as long as the person adding
-  // has all permissions granted by that group.
-  // We can't use `userHasPermission` here because we want to make sure they
-  // have ALL the permissions rather than ANY.
-  // Accounts in the "owner" group and users with the global "owner" permission
-  // are able to add any user to any group, regardless of other permissions.
-  const ownerGroup = await Groups.findOne({ name: "owner", shopId });
-  const isOwnerAccount = (
-    !!ownerGroup && contextUserAccount && contextUserAccount.groups.includes(ownerGroup._id)) ||
-    userHasPermission("reaction:legacy:shops", "owner", { shopId }); // TODO(pod-auth): update this to figure out what to do with "owner"
-
-  const hasPermissionToAddAccountToGroup = await isOwnerAccount || await userHasPermission("reaction:legacy:groups", "manage:accounts", { shopId });
+  // An account can add another account to a group as long as the account adding
+  // has `reaction:legacy:groups/manage:accounts` permissions
+  const hasPermissionToAddAccountToGroup = await userHasPermission("reaction:legacy:groups", "manage:accounts", { shopId });
```

Comment:
At this point, should we just delete this fn and pass these same args to `validationPermissions` in the two fns that call this one? More of the standard pattern.